### PR TITLE
Add redirects for wallet pages and remove from navigation

### PR DESCRIPTION
**Pull request by markmhx**
Resolves https://github.com/blockstack/docs/issues/879

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -340,7 +384,6 @@ async function redirects() {
       destination: '/storage-hubs/gaia-admin',
       permanent: true,
     },
-    { source: '/org/secureref.html', destination: '/stacks-wallet/security', permanent: true },
```
Comment (agraebe) [inline next.config.js:343]: I'm assuming it's not a good practice to remove existing URLs unless we're certain they aren't referenced anywhere. I'd suggest that we point them to the wallet homepage

**Review by agraebe — COMMENTED** [review]
looking great. I left a note irt redirect removal (form the old docs URL structure)

File: next.config.js (hunk 1 above)
Comment (markmhx) [reply next.config.js:343]: Git is tracking this as a deletion but it's actually moved and updated further up on [line 298](https://github.com/blockstack/docs/pull/882/files#diff-882b2c04b01b2e8b2cdcf1817c30ea503a8005f1c0d54cfff37053b6801fea85R298).

**Review by agraebe — APPROVED** [review]
awesome, lgtm!

**Comment by agraebe** [comment]
@markmhx could you review and resolve the conflicts? Ping me if I should help

**Pull request merged**